<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualRecolorTool</class>
 <widget class="QMainWindow" name="ManualRecolorTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual Recolor Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ImageViewer" name="originalImageViewer" native="true"/>
      </item>
      <item>
       <widget class="ImageViewer" name="recolorImageViewer" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="toolPalette" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tools</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="sampleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sample (I)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="drawButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Draw (B)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="eraseButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Erase (E)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="fillButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fill (G)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ClickableSwatch" name="selectedColorSwatch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ImageViewer" name="originalPalette" native="true"/>
       </item>
       <item>
        <widget class="ImageViewer" name="recolorPalette" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Recolor_Palette"/>
    <addaction name="actionSave_Recolor_Palette"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Images</string>
    </property>
    <addaction name="actionNo_Images_Loaded"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSample"/>
    <addaction name="actionDraw"/>
    <addaction name="actionErase"/>
    <addaction name="actionFill"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuImages"/>
  </widget>
  <action name="actionSave_Recolor_Palette">
   <property name="text">
    <string>Save Recolor Palette...</string>
   </property>
  </action>
  <action name="actionOpen_Recolor_Palette">
   <property name="text">
    <string>Open Recolor Palette...</string>
   </property>
  </action>
  <action name="actionNo_Images_Loaded">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Images Loaded</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionSample">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sample</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Draw Color...</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableSwatch</class>
   <extends>QFrame</extends>
   <header>clickableswatch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>drawButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDraw</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampleButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSample</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraseButton</sender>
   <signal>clicked()</signal>
   <receiver>actionErase</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fillButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFill</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
